src/Resources/help/MakeMigrations.txt

```diff
@@ -0,0 +1,5 @@
+The <info>%command.name%</info> command generates a new migrations:
+
+<info>php %command.full_name% doctrine:migrations:migrate</info>
+
+If the argument is missing, the command will ask for the command name interactively.
```

Comment:
not needed. But we should probably show one other example that shows the options being used.